backend/src/main/java/com/bakdata/conquery/io/external/form/ExternalTaskState.java

```diff
@@ -61,14 +61,11 @@ public boolean isValidTaskId() {
 	@JsonIgnore
 	@ValidationMethod(message = "Status is set to FAILURE, but no error information was set.")
 	public boolean isErrorInfoSet() {
-		if (status != TaskStatus.FAILURE) {
-			return true;
-		}
-
-		return error != null;
+		return (status == TaskStatus.FAILURE) ==  (error != null);
```

Comment:
My brain can't get this line right :-D. But it's also still early in the morning

Reply:
habs wieder zurückgedreht, dachte zuerst auch es sei lesbarer aber ist es nicht